jerry-core/ecma/builtin-objects/ecma-builtin-map-prototype.c

```diff
@@ -35,7 +35,7 @@
  */
 
 /**
- * The Map.prototype object's 'clear' routine
+ * The Set.prototype object's 'clear' routine
```

Comment:
I think this meant to be `Map/Set`. It is odd the comment changed for this function, but not for the other similar functions in this file (e.g.: `ecma_builtin_map_prototype_object_delete`).

Reply:
Thanks, this change was unnecessary.